recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -18,7 +18,7 @@
               <template #left>
                 <dt-icon
                   size="300"
-                  class="dt-feed-item-pill--icon d-pr8"
+                  class="dt-feed-item-pill--icon d-pr8 d-box-content"
```

Comment:
Might need to handle this in a different way, but `box-content` is required to avoid the icon shrinking when external usage has a `normalize` library that's set's pretty much everything to `border-box` that ends up in:

![image](https://github.com/dialpad/dialtone-vue/assets/5950950/233e136c-7ba6-4c47-8cf2-acb10610b826)

Reply:
We're no longer using utility classes at component level, just for consumer side.

You might want to add that to `.dt-feed-item-pill--icon` defined below.

That image is after or before applying `box-content`?

Reply:
Yes, please remove `d-pr8`, we're trying to not introduce any new utility class and we're planning to make a cleanup of every component soon.